CHANGELOG.md

```diff
@@ -3,8 +3,8 @@
 This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/en-us/documentation/articles/app-insights-release-notes-dotnet/).
 
 ## Version 2.6.0
- -[Fix: changed namespace SamplingPercentageEstimatorSettings](https://github.com/Microsoft/ApplicationInsights-dotnet/issues/727)
-
+- [Fix: changed namespace SamplingPercentageEstimatorSettings](https://github.com/Microsoft/ApplicationInsights-dotnet/issues/727)
+- [NEW: Added TryGetOperationDetail to DependencyTelemetry to facilitate advanced ITelemetryInitializer scenarios] (https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/587)
```

Comment:
this was meant for 2.6.0 stable or 2.7.0-beta1? @MS-TimothyMothra can help.

Reply:
It'd be great to explicitly mention that these new fields are NOT meant to be serialized and sent to AI backend, they only exist for facilitate TelemetryInitializers.

Reply:
Let me know if the release the note is under needs to change -- I just put it under the latest one assuming that all pending changes should be put there.  If you'd rather not release something in a stable release that hasn't been in a beta release I understand.  I'll update the release note and the comment on the set method to explicitly state that the data is not serialized.

Reply:
I think this changes would go into 2.7.0-beta1.
We had already shipped the last beta of 2.6.0 so only critical bugs would be released into 2.6.0-stable. So this change should be in beta1 of the next release, which is 2.7.0-beta1